.claude/skills/uc-benchmark-gate/SKILL.md

```
@@ -0,0 +1,14 @@
+# uc Benchmark Gate
+
+Use this skill when preparing or validating performance claims.
+
+## Standard Lanes
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)